### ISSUE TITLE
Arguments checking

### ISSUE THREAD

**Issue by josecamachop** [jogonmar/MVBatch#14]
For future versions, it would be nice to check input parameters using the same approach as in the MEDA

**Comment by jogonmar** [comment]
Agreed! In the next release, I will ensure that the input parameter check is standardized in all the functions. To avoid forgetting this item, I will create a text file "Improvements.txt" to list the actions to take for the next release for both sides.

**Comment by josecamachop** [comment]
Ok, but the issues are our list of improvements.
